server.go

```diff
@@ -43,6 +45,17 @@ func main() {
 	reqLog := flag.Bool("reqlog", false, "Enables request logging -- do NOT use in production")
 	flag.Parse()
 
+	err := os.Mkdir("/etc/ielogs", 0755)
+	if err != nil && !os.IsExist(err){
+		fmt.Println("Error creating log directory:" + err.Error())
+	}
+
+	lf, err := os.Create("/etc/ielogs/ie.log")
```

Comment:
`os.Create` will create a new file or _truncate_ and existing file.  That means that if the service is restarted, all the existing logs in the file will be deleted.  I don't think that's what we want.  We want to create it if it doesn't exist, otherwise we want to append it.